<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1342</width>
    <height>1097</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Control Predict</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="PlotWidget_trajectory" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>431</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>T</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_T">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>40</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>8.4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_compute">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>40</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Compute</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="PlotWidget_P" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>431</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="PlotWidget_N" native="true">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>350</y>
      <width>431</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="PlotWidget_nu_12" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>690</y>
      <width>431</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="PlotWidget_U" native="true">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>690</y>
      <width>431</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="PlotWidget_PT" native="true">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>350</y>
      <width>431</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="PlotWidget_nu_3" native="true">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>690</y>
      <width>431</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="PlotWidget_theta" native="true">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>431</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1342</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
